### ISSUE TITLE
Cannot invoke "String.getBytes()" because "payload" is null

### ISSUE THREAD

**Issue by tobias-zeptio** [eclipse/ditto#1503]
Reading section 3.2 in https://www.amqp.org/sites/amqp.org/files/amqp.pdf the body of an AMQP 1.0 message is defined as follows:

- The body consists of either: one or more data sections, one or more amqp-sequence sections,
or a single amqp-value section.

A valid amqp value is null, which the ditto-connectivity AmqpConsumerActor doesn't seem to handle in the `extractPayloadFromMessage` method.

I think adding a null check and returning a ExternalMessageBuilder with null values should be valid.

The use-case here is handling retrieve commands in the payload-mapper, in our case the header contains which property to read.

**Comment by tobias-zeptio** [comment]
Looks like UnmodifiableExternalMessageBuilder accepts null as payload so I think this would suffice, so no change needed there.

I'm looking into how to get the body from the `JmsMessage`, but it seems to always return `null` which is strange.

```
    @Override
    public boolean isBodyAssignableTo(@SuppressWarnings("rawtypes") Class target) throws JMSException {
        return true;
    }

    @Override
    public final <T> T getBody(Class<T> asType) throws JMSException {
        if (isBodyAssignableTo(asType)) {
            return doGetBody(asType);
        }

        throw new MessageFormatException("Message body cannot be read as type: " + asType);
    }

    protected <T> T doGetBody(Class<T> asType) throws JMSException {
        return null;
    }
```

We use qpid-jms-client in another project, and there is straight up returns the body but that uses `JmsMessageFacade` instead of `JmsMessage`.

**Comment by tobias-zeptio** [comment]
Resolved by #1505
